FieldSupplyAIDriver.lua

```diff
@@ -164,29 +167,20 @@ function FieldSupplyAIDriver:getSiloSelectedFillTypeSetting()
 	return self.vehicle.cp.settings.siloSelectedFillTypeFieldSupplyDriver
 end
 
-function FieldSupplyAIDriver:isOverloadingTriggerCallbackEnabled()
-	return true
-end
-
-function FieldSupplyAIDriver:isUnloadingTriggerCallbackEnabled()
-	return true
-end
-
 --Augerwagons handling
 --Pipe callback used for augerwagons to open the cover on the fillableObject
 function FieldSupplyAIDriver:unloadingTriggerCallback(superFunc,triggerId, otherId, onEnter, onLeave, onStay, otherShapeId)
 	local rootVehicle = self:getRootVehicle()
-	if courseplay:isAIDriverActive(rootVehicle) then 
-		if not rootVehicle.cp.driver:isOverloadingTriggerCallbackEnabled() then
-			return superFunc(self,triggerId, otherId, onEnter, onLeave, onStay, otherShapeId)
-		end
+	if courseplay:isAIDriverActive(rootVehicle) and rootVehicle.cp.driver.triggerHandler.validFillTypeUnloadingAugerWagon then 
```

Comment:
Don't fully understand the meaning of validFillTypeUnloadingAugerWagon but would be great if we could avoid accessing triggerHandler internals directly

Reply:
Maybe this function cloud be in the TriggerHandler as well?